packages/thirdweb/src/pay/convert/type.ts

```diff
@@ -27,22 +27,34 @@
 export type SupportedFiatCurrency = (typeof CURRENCIES)[number] | (string & {});
 
 export function getFiatSymbol(showBalanceInFiat: SupportedFiatCurrency) {
-  switch (showBalanceInFiat) {
-    case "USD":
-      return "$";
-    case "CAD":
-      return "$";
-    case "GBP":
-      return "£";
-    case "EUR":
-      return "€";
-    case "JPY":
-      return "¥";
-    case "AUD":
-      return "$";
-    case "NZD":
-      return "$";
-    default:
-      return "$";
+  if (currencySymbol[showBalanceInFiat]) {
+    return currencySymbol[showBalanceInFiat];
   }
+  return "$";
 }
+
+const currencySymbol: Record<SupportedFiatCurrency, string> = {
```

Comment:
do we actually use this? i think we can just use the built in currency formatter no?

Reply:
We can do that, but it's kinda more complicated because that requires creating a new instance for Intl.NumberFormat and then formatting the value and then slicing the symbol part of. This is much simpler imo

Reply:
I already have a utility for this i think? Check out formatCurrency() 

Important to centralize these formatters/display helpers imo

Reply:
Yeah, I saw that, it formats the whole value as I mentioned above - but then we need to extract the symbol out of it because we only need a symbol here